src/TMedia.cpp

```diff
@@ -202,9 +202,35 @@ void TMedia::parseGMCP(QString& packageMessage, QString& gmcp)
         TMedia::parseJSONForMediaPlay(json);
     }
 }
+
+bool TMedia::purgeMediaCache()
+{
+    QString mediaPath = mudlet::getMudletPath(mudlet::profileMediaPath, mpHost->getName());
+    QDir mediaDir(mediaPath);
+
+    if (!mediaDir.mkpath(mediaPath)) {
+        qWarning() << QStringLiteral("TMedia::purgeMediaCache() WARNING - Not able to reference directory: %1").arg(mudlet::getMudletPath(mudlet::profileMediaPath, mpHost->getName()));
+        return false;
+    }
+
+    TMedia::stopAllMediaPlayers();
```

Comment:
Do we need the class prefix here? Or is there more than one `stopAllMediaPlayers()` method around?

Reply:
I removed it for you @SlySven.